src/SDKs/DataMigration/DataMigration.Tests/DataMigration.Tests.csproj

```diff
@@ -2,13 +2,13 @@
   <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
     <PackageId>DataMigration.Tests</PackageId>
-    <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <AssemblyName>DataMigration.Tests</AssemblyName>
     <Description>DataMigration.Tests Class Library</Description>
+    <AssemblyName>DataMigration.Tests</AssemblyName>
+    <VersionPrefix>1.0.0</VersionPrefix>    
```

Comment:
Please replace this with `<Version>1.0.0</Version>`

Reply:
Done